README.md

```diff
@@ -71,20 +71,14 @@ In these cases, you can swap out the URL for the relevant frontend (such as draf
 Sometimes things go wrong or some investigation is needed. As govuk-docker is just a bunch of docker config and a CLI wrapper, it's still possible to use all the standard docker commands to help fix issues and get more info e.g.
 
 ```
-# make sure govuk-docker is up-to-date
```

Comment:
I'm not sure if this is so obvious. This is in a section about troubleshooting which (hopefully) doesn't happen very often. If people have a govuk-docker and it's working, I imagine they're unlikely to do `git pull` very often and it may not occur to them to be the first thing to do.

Reply:
Fair enough - it's back 👍